pkg/commands/workload_get.go

```diff
@@ -139,6 +138,13 @@ func (opts *WorkloadGetOptions) Exec(ctx context.Context, c *cli.Config) error {
 		}
 	}
 
+	c.Printf("\n")
```

Comment:
Stick to the printing pattern in get file
- add a new line if you are adding a new block. So `c.Printf("\n")` should be within the if block
- do not add new line at the end of block. 

instead of 
```
c.Infof("Supply Chain resources not found\n")
```

do 
```
c.Infof("Supply Chain resources not found")
```

Reply:
Line 141 `c.Printf("\n")` is added because either way, if there are or there are not resources, there is a message that's going to be printed (either the table with the resources or `not resources found`). It would have to be added twice if it's put on each part of the if statement.

Reply:
thanks